prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -27,10 +27,25 @@ asap_accepted_audiences = { "{{ join "\",\"" (splitList "," .Env.JWT_ACCEPTED_AU
 {{ end }}
 
 {{ if $ENABLE_XMPP_WEBSOCKET }}
+consider_bosh_secure = true;
```

Comment:
Do we need this for websockets only? I think it would be great to remove the `if`.

Reply:
Yeah, let's drop it.